programs/models.py

```diff
@@ -1119,9 +1131,9 @@ class UrgentNeed(models.Model):
     notification_message = models.ForeignKey(
         Translation,
         related_name="urgent_need_notification_message",
-        blank=False,
-        null=False,
-        on_delete=models.PROTECT,
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
I don't think I follow these changes. Can you explain?

Reply:
Oh god, thanks for catching this. I think this was just a bad git shuffle that these found their way into review.

These values were from holdovers from a now-fixed version of the branch I based this off of. They should not change.

I'll remove them!